outlines/models/vllm.py

```diff
@@ -134,6 +140,10 @@ def generate(
             lora_request=self.lora_request,
             use_tqdm=use_tqdm,
         )
+
+        if original_output:
+            return results
```

Comment:
This part won't pass coverage checks because it isn't tested. Could you add a test here: https://github.com/dottxt-ai/outlines/blob/main/tests/generate/test_integration_vllm.py